fbfmaproom/data-conversion-scripts/oromia/bad-years-v3-3season.py

```diff
@@ -0,0 +1,43 @@
+import cftime
+import datetime
+import pandas as pd
+import xarray as xr
+import sys
+import os
+
+dirout='/data/aaron/fbf-candidate/'
+regionsdic = {
+        "oromia": {"file": "/data/aaron/fbf-candidate/original-data/oromia/bad_years.csv"}
+}
+
+seasons=["DJF","JFM","FMA","MAM","AMJ","MJJ","JJA","JAS","ASO","SON","OND","NDJ"]
```

Comment:
We have used one- and four-month seasons occasionally in the AA project. This is fine for now for Oromia, but if we're going to generalize this, I think we need a different representation of the season, e.g. Dec-Feb like in Ingrid.

Reply:
OK, but this would even be in the file name, right? Meaning the final file would be called "southern-oromia-bad-years-v3-Oct-Dec.zarr"

Reply:
Not necessarily, since the filename is arbitrary, but it wouldn't hurt to be consistent. Currently 

(Let's not delay getting the Oromia data into the tool while we discuss standardization for the longer term.)